helper/src/configpresets/baselines.json

```diff
@@ -20,51 +20,78 @@
+                                    "description": "East-West traffic control",
+                                    "linksrc": "https://docs.microsoft.com/en-gb/azure/aks/use-network-policies"
+                                },
+                                {
+                                    "description": "Store Kubernetes Secrets in Azure KeyVault,",
+                                    "linksrc": "https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver"
                                 }
                             ]
                         },
                         "imageSrc": "https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/containers/aks/images/secure-baseline-architecture.svg",
                         "values": {
                             "cluster": {
                                 "enable_aad": true,
-                                "AksDisableLocalAccounts": false,
+                                "AksDisableLocalAccounts": true,
                                 "apisecurity": "none",
                                 "autoscale": true,
                                 "agentCount": 2,
-                                "maxCount": 10,
+                                "maxCount": 5,
                                 "upgradeChannel": "none",
-                                "DefenderForContainers": true
+                                "DefenderForContainers": true,
+                                "AksPaidSkuForSLA": true,
+                                "SystemPoolType": "Standard",
+                                "availabilityZones": "yes"
                             },
                             "addons": {
                                 "networkPolicy": "azure",
-                                "registry": "Basic",
+                                "registry": "Premium",
                                 "azurepolicy": "audit",
-                                "ingress": "none",
+                                "azurePolicyInitiative": "Restricted",
+                                "ingress": "traefik",
                                 "appgwKVIntegration": false,
                                 "monitor": "aci",
-                                "csisecret": "none"
+                                "csisecret": "akvNew",
+                                "acrUntaggedRetentionPolicy": 15,
+                                "acrUntaggedRetentionPolicyEnabled": true
                             },
                             "net": {
-                                "vnetprivateend": false,
-                                "vnet_opt": "default",
-                                "afw": true
+                                "vnetprivateend": true,
+                                "vnet_opt": "custom",
+                                "afw": true,
+                                "azureFirewallSku": "Premium",
+                                "nsg": true
```

Comment:
Do both RI's have nsg's?
I thought NSG was unique just to regulated.

Reply:
No secure baseline standard definitely has NSG's